src/transformers/configuration_xlm_prophetnet.py

```diff
@@ -0,0 +1,36 @@
+# coding=utf-8
+# Copyright 2020 The Microsoft Authors and The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" XLM-ProphetNet model configuration """
+
+
+import logging
+
+from .configuration_prophetnet import ProphetNetConfig
+
+
+logger = logging.getLogger(__name__)
+
+XLM_PROPHETNET_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "microsoft/xprophetnet-large-wiki100-cased": "https://s3.amazonaws.com/models.huggingface.co/bert/microsoft/xprophetnet-large-wiki100-cased/config.json",
+}
+
+
+class XLMProphetNetConfig(ProphetNetConfig):
+    """
+    This class overrides :class:`~transformers.RobertaConfig`. Please check the
```

Comment:
```suggestion
    This class overrides :class:`~transformers.ProphetNetConfig`. Please check the
```
Are the default the same? If not, the docstring should be copied from `ProphetNetConfig` and adapted (and the subclass should be avoided altogether).